code/modules/mob/new_player/login.dm

```diff
@@ -30,6 +30,23 @@
 	callHook("mob_login", list("client" = client, "mob" = src))
 
 	new_player_panel()
+	
+	spawn(30)
+		// Annoy the player with polls.
+		establish_db_connection()
+		if(dbcon.IsConnected() && client.can_vote())
```

Comment:
`log_client_to_db` runs after `mob/Login` - so this first `can_vote` will runtime, as `player_age` is `"--"`